spec/java_buildpack/component/versioned_dependency_component_spec.rb

```diff
@@ -48,6 +50,13 @@
       expect(component.detect).to eq("stub-versioned-dependency-component=#{version}")
     end
 
+    it 'downloads and installs InstallAnywhere (tm) BIN file in the sandbox',
```

Comment:
This functionality is IBM JRE specific and therefore shouldn't be tested as part of this class.